Makefile

```diff
@@ -1,16 +1,34 @@
-all: env test packages
+all: env packages doc test
+all-docker: env-docker packages-docker doc-docker test-docker
 
 clean:
-	bin/clean.sh
+	@bash bin/clean.sh
+
+distclean: clean
+	@bash bin/distclean.sh
 
 env:
-	bin/env.sh
+	@bash bin/env.sh
```

Comment:
why do these all need to be prefixed with bash? isn't having them all use a hashbang and be executable enough?